packages/typespec-ts/test/modularUnit/scenarios/models/apiVersion/apiVersionAsFixedEnum.md

```diff
@@ -59,15 +58,11 @@ import {
 
 export function _fooSend(
   context: Client,
-  apiVersion: Versions,
```

Comment:
header api version parameter is missing?

Reply:
should be expected as there's versioned decorator and apiVersion will be lifted to the client level no matter if apiVersion exists in all operations and if the apiVersion parameter is a query/header parameter by tcgc. https://github.com/Azure/typespec-azure/issues/1595